dhis-2/dhis-api/src/main/java/org/hisp/dhis/changelog/ChangeLogType.java

```diff
@@ -25,14 +25,14 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.hisp.dhis.common;
+package org.hisp.dhis.changelog;
 
 /**
  * This class is deprecated in favor of new async auditing solution, do not use.
  *
  * @author Halvdan Hoem Grelland
  */
-public enum AuditType {
+public enum ChangeLogType {
   CREATE,
   UPDATE,
   DELETE,
```

Comment:
Is this class truly separate from the other type of audits we are trying to separate from? I see `READ`, `SEARCH` which sound more like the traces of every action i.e. audit and not like changelog 🤔 

If this class is used for both purposes does it make sense to split it? Like the approach we used so far with old/new tracker. Duplicate first, then rename.

Reply:
That is a good point. The "real" [audit](https://github.com/dhis2/wow-backend/blob/master/guides/auditing.md) is already in place, but it uses another [AuditType](https://github.com/dhis2/dhis2-core/blob/91df59ab9a47d3975ef63108b22882706e4c7219/dhis-2/dhis-api/src/main/java/org/hisp/dhis/audit/AuditType.java#L33), and it is a bit more complex flow that uses JMS.
 @mortenoh   knows more about auditing in any case, so I guess he can answer more specific questions about the semantics of particular choices.

Reply:
Ok, got it thank you! We change `org.hisp.dhis.common.AuditType` => `ChangeLog` while
`org.hisp.dhis.audit.AuditType` remains as the "real" audit type 😅 .

Reply:
Yes, @teleivo, I agree that would need some refinement. Likewise, I am unsure that everything we are moving in the current `/audits` is an actual change log. So, we'll need to make some clean when making the new endpoints.